spk/rest-server/src/service-setup.sh

```diff
@@ -0,0 +1,50 @@
+
+PATH="${SYNOPKG_PKGDEST}/bin:${PATH}"
+GROUP="sc-rest-server"
+
+INST_ETC="/var/packages/${SYNOPKG_PKGNAME}/etc"
+INST_VARIABLES="${INST_ETC}/installer-variables"
+
+# Reload wizard variables stored by postinst
+if [ -r "${INST_VARIABLES}" ]; then
+    # we cannot source the file to reload the variables, when values have special characters like <, >, ...
+    for _line in $(cat "${INST_VARIABLES}"); do
+        _key="$(echo ${_line} | awk -F'=' '{print $1}')"
+        _value="$(echo ${_line} | awk -F'=' '{print $2}')"
+        declare "${_key}=${_value}"
+    done
+fi
+
+ARGS=""
+
+if [ "${WIZARD_APPEND_ONLY}" == "true" ]; then
+    ARGS="${ARGS} --append-only"
+fi
+if [ "${WIZARD_PRIVATE_REPOS}" == "true" ]; then
+    ARGS="${ARGS} --private-repos"
+fi
+if [ "${WIZARD_PROMETHEUS}" == "true" ]; then
+    ARGS="${ARGS} --prometheus"
+
+    if [ "${WIZARD_PROMETHEUS_NO_AUTH}" == "true" ]; then
+        ARGS="${ARGS} --prometheus-no-auth"
+    fi
+fi
+
+PORT=8500
```

Comment:
You can ommit to need the port definition here.

Add `SERVICE_PORT = 8500` to the Makefile and you will have the variable `SERVICE_PORT` with this port in service-setup.sh.

I suppose the there is no web UI available but the REST API only?
So you need to define  `NO_SERVICE_SHORTCUT` to avoid a link in DSM Ui, 
and you must remove `ADMIN_PORT = 8500` in the Makefile (that would create a link in the DSM Package Center to the non existing web page)

Reply:
Fixed in https://github.com/SynoCommunity/spksrc/pull/5550/commits/d8ea928913b6679d8e72201c7b8e9f61c764743d